src/components/image-tooltip/ImageTooltipContent.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../styles/variables';
+
+.bdl-ImageTooltipContent-image {
+    display: flex;
+
+    img {
```

Comment:
lets avoid using tag selectors, since itll match on every image on the page during parsing